packages/website/pages/index.js

```diff
@@ -9,15 +9,15 @@ import Link from 'next/link'
 import { FAQ } from './faq'
 
 export async function getStaticProps() {
-  const logos = await fs.readdirSync('public/images/marketplace-logos')
+  const logos = fs.readdirSync('public/images/marketplace-logos')
   // make opensea be the first logo
   logos.sort((a, b) =>
     a.includes('opensea') ? -1 : b.includes('opensea') ? 1 : 0
   )
 
   return {
     props: {
-      needsUser: false,
+      needsUser: true,
```

Comment:
It does?

Reply:
yes and no, it allows us to skip one magic round-strip in the callback route and avoid the layout shift of landing in the homepage logged out for a bit and re-use session after the request.